src/api/projects/post.ts

```diff
@@ -26,20 +26,15 @@ module.exports = (req: Request, res: Response) => {
         return;
     }
 
-
     (async () => {
-        const user = await GetDiscordUser(req.body.accessToken).catch((err) => genericServerError(err, res));
-        if (!user) {
-            res.status(401);
-            res.end(`Invalid accessToken`);
-            return;
-        }
-        
-        let discordId = (user as IDiscordUser).id;
+        let discordId = await GetDiscordToken(req,res);        
 
         submitProject(body, discordId)
-            .then(results => {
-                res.end("Success");
+            .then(() => {
+                res.status(200);
+                res.json(JSON.stringify({
```

Comment:
So although this _works_, you don't need to `JSON.stringify` an object when using `res.json` as it's done as part of the jsonification. I'll be opening a PR at some point to fix this across the project

Reply:
good catch, it was from some previous code :)